asserts/findwildcard.go

```diff
@@ -101,11 +117,67 @@ func findWildcardDescend(top, current string, descendantWithWildcard []string, f
 			return fmt.Errorf("findWildcard: invoked with malformed wildcard: %v", err)
 		}
 		if ok {
-			err = findWildcardDescend(top, filepath.Join(current, name), descendantWithWildcard[1:], foundCb)
+			err = findWildcardDescend(top, filepath.Join(current, name), descendantWithWildcard[1:], seqnum, foundCb)
 			if err != nil {
 				return err
 			}
 		}
 	}
 	return nil
 }
+
+func findWildcardSequence(top, current, seqWildcard string, descendantWithWildcard []string, seqnum int, foundCb func(relpath []string) error) error {
+	filter := func(i int) bool { return true }
+	if seqnum != -1 {
+		if seqWildcard == "#>" {
+			filter = func(i int) bool { return i > seqnum }
+		} else {
+			filter = func(i int) bool { return i < seqnum }
+		}
+	}
+
+	d, err := os.Open(current)
+	if os.IsNotExist(err) {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
+	defer d.Close()
+	var seq []int
+	for {
+		names, err := d.Readdirnames(100)
```

Comment:
Nothing wrong with that, but just curious; is there a concern about high volume of names & reading them all in one go?

Reply:
not really, but we need to make seq anyway so mostly trying not to allocate the full things twice